flink-core/src/main/java/org/apache/flink/configuration/SecurityOptions.java

```diff
@@ -132,6 +132,22 @@ public class SecurityOptions {
                     .withDescription(
                             "The time period when keytab login happens automatically in order to always have a valid TGT.");
 
+    @Documentation.Section(Documentation.Sections.SECURITY_AUTH_KERBEROS)
+    public static final ConfigOption<Duration> KERBEROS_TOKENS_RETRY_WAIT =
+            key("security.kerberos.tokens.retry-wait")
```

Comment:
I think a better name would be `security.kerberos.tokens.obtain.retry.backoff` or `security.kerberos.tokens.retry.backoff`

Reply:
Renamed to `security.kerberos.tokens.retry.backoff`.